<?xml version="1.0" encoding="utf-8" ?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="Docutils 0.8.1: http://docutils.sourceforge.net/" />
<title>Peter Wilton</title>
<link rel="stylesheet" href="pwilton.css" type="text/css" />
<link rel="icon" href="favicon.ico" />
</head>
<body>

<div id="header"> </div>

<div class="document" id="peter-r-wilton">


<h2 class="title">Peter R. Wilton</h2>

<p>I am a computational biologist and population geneticist, working to turn
observations of genetic variation within populations into insights about
evolutionary and demographic history. From July 2016 to April 2019 I was a
postdoc with <a class="reference external"
    href="https://www.nielsenlab.org/">Rasmus
Nielsen</a> in the Departments of <a class="reference external"
href="https://ib.berkeley.edu/">Integrative Biology</a> and <a
href="https://statistics.berkeley.edu/">Statistics</a> at UC Berkeley. Before
that, I was a PhD student studying theoretical population genetics with <a
    href="https://wakeleylab.oeb.harvard.edu/">John Wakeley</a> at Harvard
University. During Summer 2019, I hiked 1750 miles of the <a
href="https://www.pcta.org/">Pacific Crest Trail</a>, and I am currently
looking to transition into a role in biotech.
</p>

<p>You can reach me at <a
href="mailto:peterrwilton&#64;gmail.com">peterrwilton&#64;gmail.com</a></p>

<hr>

<p>As a postdoc, I worked on multiple projects in applied population genetics,
statistical genetics, and bioinformatics.</p>

<p>My primary project involved designing population-genetic inference methodology
for understanding mitochondrial genetic variation within the body and between
mother and offspring (i.e., mitochondrial heteroplasmy). The mitochondria
transmitted between mother and offspring&mdash;and those within an
individual&mdash;can be thought of as a network of mitochondrial populations
related by an ontogenetic phylogeny. Each branch of this phylogeny represents a
different stage of life, during which the relative frequencies of different
mitochondrial genotypes can change. Adapting approaches used in population
phylogenetics, I have developed a method that uses heteroplasmy frequencies in
different tissues to generate insight into the ontogenetic dynamics of
mitochondrial inheritance and proliferation. This project is a collaboration
with <a href = "http://www.bx.psu.edu/makova_lab/"> Kateryna Makova</a> at Penn
State University, whose lab has generated a large dataset of heteroplasmy
frequencies in multi-generation human pedigrees.</p>

<p>Another project involves investigating signals of diminishing-returns or
synergistic epistatsis in the <a href="https://www.ukbiobank.ac.uk/">UK
Biobank</a> data, adding non-linearity terms to standard additive polygenic
score models. This work is ongoing.</p>

<p>One current limitation of current sequencing technologies is that they struggle
to identify low-frequency variants in somatic DNA libraries. In collaboration
with my postdoctoral advisor Rasmus Nielsen, I have designed and implemented a
novel empirical Bayesian approach to distinguishing sequencing error from true
low-frequency variants. See the project's
<a href="https://github.com/ammodramus/ebsom">GitHub page</a> for details.</p>

<hr>

<p>
Previously, as a graduate student with
<a href="https://wakeleylab.oeb.harvard.edu/"> John Wakeley</a>
at Harvard University's Department of
<a href="https://oeb.harvard.edu/">Organismic and Evolutionary
Biology</a>, I studied various topics in theoretical population genetics,
mostly related to the coalescent process. These topics included

<ul class="arabic simple">
<li>sequentially Markov coalescent models associated inference methods,</li>
<li>population pedigrees and their effect on the coalescent process and genetic
variation, and</li>
<li>coalescent models of long segments of identity by descent.</li>
</ul>

<div class="section" id="publications">

<h3>Publications</h3>

<p>
<strong>Wilton, P.</strong>, Zaidi, A., Makova, K., and Nielsen, R. <a
    href="http://www.genetics.org/content/early/2018/01/17/genetics.118.300711">
    A population phylogenetic view of mitochondrial heteroplasmy</a>. Genetics,
    2018. (<a
    href="https://www.biorxiv.org/content/early/2017/10/17/204479">bioRxiv</a>)
</p>

<p>
Ng, N.S.R., <strong>Wilton, P.R.</strong>, Prawiradilaga, D.M., Tay, Y.C.,
Indrawan, M., Garg, K.M., and Rheindt, F.E.
<a href="http://www.sciencedirect.com/science/article/pii/S1055790316304080">
The effects of Pleistocene
climate change on biotic differentiation in a montane songbird clade from
Wallacea</a>. Molecular Phylogenetics and Evolution, 2017.
</p>

<p>
<strong>Wilton, P.R.</strong>, Baduel, P., Landon, M.M., and Wakeley, J.
<a href="http://www.sciencedirect.com/science/article/pii/S0040580917300084">
    Population structure and coalescence in pedigrees: comparisons to the
    structured coalescent and a framework for inference.</a>
Theoretical Population Biology, 2017. (<a
href="http://biorxiv.org/content/early/2016/05/24/054957">bioRxiv</a>)
</p>

<p>
Wakeley J., <strong>Wilton P.R.</strong>
<a href="http://www.sciencedirect.com/science/referenceworks/9780128004265#ancpt0065">
    Coalescent and models of identity by descent. In: Encyclopedia of
    Evolutionary Biology.</a> Vol 1. Academic Press, Oxford, 2016: pp. 287-292.
(<a href="files/wakeleyandwilton2016enc.pdf">pdf</a>)
</p>

<p>
Wakeley, J., King, L., and <strong>Wilton, P.R.</strong>
<a href="http://www.pnas.org/content/113/29/7994.abstract">
    Effects of the population pedigree on genetic signatures of historical
    demographic events.</a> PNAS, 2016.
</p>

<p>Palamara, P.F., Francioli, L., <strong>Wilton, P.R.</strong>, Genovese, G., 
Gusev, A., Finucane, H., Sankararaman, S., Sunyaev, S., Debakker, P., Wakeley,
J., Pe'er, I., Price, A.L. <a
href="http://www.sciencedirect.com/science/article/pii/S0002929715004085">Leveraging distant
relatedness to quantify human mutation and gene conversion rates.</a> American
Journal of Human Genetics 2015. (<a href="http://biorxiv.org/content/early/2015/06/16/020776">bioRxiv</a>)</p> 

<p><strong>Wilton, P.R.</strong>, Carmi, S., and Hobolth, A. <a class="reference external" href = "http://www.genetics.org/content/200/1/343">The SMC’ is a highly accurate
    approximation to the ancestral recombination graph. </a>
Genetics 2015.
(<a class="reference external" href="http://arxiv.org/abs/1501.02847v2">arXiv</a>)</p>

<p>Rheindt, F.E., Prawiradilaga, D.M., Suparno, S., Ashari, H., and <strong>Wilton,
    P.R.</strong> <a class="reference external"
    href="http://e-journal.biologi.lipi.go.id/index.php/treubia/article/view/458">
    New and significant island records, range extensions and elevational
    extensions of birds in eastern Sulawesi, its nearby satellites, and
    Ternate</a>. Treubia 2014.</p>

<p>Carmi, S., <strong>Wilton, P.R.</strong>, Wakeley, J., and Pe’er, I. <a
    class="reference external"
    href="http://www.sciencedirect.com/science/article/pii/S0040580914000562">A
    renewal theory approach to IBD sharing</a>.  Theoretical Population Biology
2014. (<a href="http://arxiv.org/abs/1403.1325">arXiv</a>) </p>

<p>Rheindt, F.E., Fujita, M.K., <strong>Wilton, P.R.</strong>, Edwards, S.V.,
2014.  <a class="reference external"
    href="http://sysbio.oxfordjournals.org/content/63/2/134">Introgression and
    phenotypic assimilation in <em>Zimmerius</em> flycatchers (Tyrannidae):
    Population genetic and phylogenetic inferences from genome-wide SNPs</a>.
Systematic Biology 2014.</p>

<p><strong>Wilton, P.R.</strong>, Sloan, D.B., Logsdon Jr, J.M., Doddapaneni,
H., and Neiman, M. <a class="reference external"
    href="http://onlinelibrary.wiley.com/doi/10.1111/1755-0998.12051/abstract">Characterization
    of transcriptomes from sexual and asexual lineages of a New Zealand snail
    (<em>Potamopyrgus</em> <em>antipodarum</em>)</a>.  Molecular Ecology
Resources 2013.</p>

<p>Neiman, M., Larkin, K., Thompson, A.R., and <strong>Wilton, P.</strong> <a
    class="reference external"
    href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3375405/">Male offspring
    production by asexual <em>Potamopyrgus</em> <em>antipodarum</em>, a New
    Zealand snail</a>. Heredity 2012.</p>

<p>Willacker, J.J., von Hippel, F.A., <strong>Wilton, P.R.</strong>, and
Walton, K.M.  <a class="reference external"
    href="http://onlinelibrary.wiley.com/doi/10.1111/j.1095-8312.2010.01531.x/full">Classification
    of threespine stickleback along the benthic–limnetic axis</a>.  Biological
Journal of the Linnean Society 2010.</p>

<p><i>
See also my Google Scholar <a
    href="https://scholar.google.com/citations?hl=en&user=Dj8y9C8AAAAJ&view_op=list_works&sortby=pubdate">profile</a>
and my PhD
<a href="https://dash.harvard.edu/bitstream/handle/1/33493608/WILTON-DISSERTATION-2016.pdf?sequence=1">
dissertation</a>.</i>
</p>


</div>

<hr>

<div class="section" id="software">

<h2>Software (see <a href="https://www.github.com/ammodramus/">GitHub</a></h2>

<dl class="docutils">

    <dt><a class="reference external" href="https://github.com/ammodramus/mope">mope</a></dt>
    <dd> Infer dynamics of mitochondrial heteroplasmy throughout many life
    stages using population genetics and phylogenetics.</dd>

    <dt><a class="reference external" href="https://github.com/ammodramus/coalped">coalped</a></dt>
    <dd>Simulates coalescence through randomly-generated population
    pedigrees. Coalped can accommodate variable population sizes,
    multiples demes, variable migration rates, and other features. It can
    produce coalescence-time distributions and infinite-alleles and
    infinite-sites mutation data in a variety of formats.</dd>

    <dt>
    <a href="https://github.com/ammodramus/infsites">Infinite-sites likelihood calculator</a></dt>
    <dd>Calculates sampling probabilities under the infinite-sites mutation
    model without recombination in a single- or two-deme setting. Uses the
    algorithm of
    <a href="http://ieeexplore.ieee.org/xpl/articleDetails.jsp?arnumber=5383348">
    Wu (2010)</a>.</dd>

</dl>
</div>

<div class="section" id="teaching">

<h2>Teaching</h2>

<dl class="docutils">

<dt>Statistics for Biology (Fall 2014)</dt>

<dd>I was a teaching assistant for OEB 153, <a class="reference external"
    href="http://isites.harvard.edu/course/colgsas-49559">Statistics for
    Biology</a>. For the course I created a sequence of biology-flavored labs
    introducing the <a class="reference external"
    href="http://www.r-project.org/">R</a> statistical computing environment.
    The labs are available on <a class="reference external"
    href="https://github.com/ammodramus/R-labs">GitHub</a>.</dd>

<dt>Coalescent Theory (Fall 2012, Fall 2015)</dt>
<dd>I was a teaching assistant for OEB 252, Coalescent Theory, a graduate-level
course.</dd>

</dl>
</div>

<div class="section" id="cv">

<h2>CV</h2>

<p>Available <a href="files/cv.pdf">here.</a></p>

</div>

<div class="section", id="wf">

<h2>Interactive Wright-Fisher process</h2>

<p>An experiment with JavaScript&mdash;try it <a href="wf.html" class="reference external">here</a>.</p>

<p><em>Last updated Jan 2018</em></p>
</body>
</html>
